inscriptions/ISic000483.xml

```diff
@@ -186,23 +192,36 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><name>Furiae</name> <g ref="#interpunct">·</g> <name>Sa
+                    <lb n="2" break="no"/>biniae</name> <g ref="#interpunct">·</g> <name>Tran
```

Comment:
the second I of Sabiniae is marked in CIL as restored (in small italics in CIL), so put it in a 'supplied' element.